reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -24,7 +24,7 @@ import ch.frankel.slf4k.*
 import com.here.ort.model.ScanRecord
```

Comment:
Commit message nits:
- "the the"
- "hyerplink"

Reply:
Done.